gluecodium/src/main/resources/templates/cpp/common/Optional.mustache

```diff
@@ -65,16 +65,11 @@
 #endif
 
 #if !defined(GLUECODIUM_OPTIONAL_HEADER) && !defined(GLUECODIUM_OPTIONAL_TYPE)
```

Comment:
Description above should be changed slightly, this should be removed

> fallback to a built-in optional implementation otherwise.

Reply:
Done